### PR TITLE
Front-end (angular) tests

### PR DESCRIPTION
**Pull request by qmonmert**
Add some front-end (angular) tests to improve code coverage

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by qmonmert** [comment]
@pascalgrimaud do you think that you can merge this PR ?

File: generators/client/templates/angular/src/test/javascript/spec/app/core/login/login-modal.service.spec.ts.ejs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                // GIVEN
+                const mockModalRef: MockNgbModalRef = new MockNgbModalRef();
+                spyOn(modalService, 'open').and.returnValue(mockModalRef);
+
+                // WHEN
+                service.open();
+
+                // THEN
+                expect(modalService.open).toHaveBeenCalled();
+                });
```
Comment (pascalgrimaud) [inline generators/client/templates/angular/src/test/javascript/spec/app/core/login/login-modal.service.spec.ts.ejs]: indent

File: generators/client/templates/angular/src/test/javascript/spec/app/core/login/login-modal.service.spec.ts.ejs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+                it('Should call open method for NgbModal one time when open method is called twice', () => {
```
Comment (pascalgrimaud) [inline generators/client/templates/angular/src/test/javascript/spec/app/core/login/login-modal.service.spec.ts.ejs]: indent

**Comment by pascalgrimaud** [comment]
@qmonmert : thanks for your PR and LGTM. But I'm not a front expert, so I'd prefer waiting a quick review from @vishal423  or @wmarques

**Comment by pascalgrimaud** [comment]
Oh didn't see the approuval from @chrisdns 
So then, we can merge this -> @qmonmert : I'm fixing the 2 minor comments in another PR

**Pull request merged**